cms/envs/common.py

```diff
@@ -1115,3 +1120,25 @@
     'options': {},
 }
 PROCTORING_SETTINGS = {}
+
+
+############################ OAUTH2 Provider ###################################
+
+# OpenID Connect issuer ID. Normally the URL of the authentication endpoint.
+OAUTH_OIDC_ISSUER = 'https://www.example.com/oauth2'
```

Comment:
Okay, I/we need to open a DevOps ticket to make sure this setting is overridden with the correct value.